README.md

```diff
@@ -14,10 +14,13 @@ The `local_lytix` plugin is a comprehensive tool designed to enhance the digital
 
 ## Requirements
 
-- Moodle Version: 4.1+
-- PHP Version: 7.4+
-- Supported Databases: MariaDB, PostgreSQL
-- Supported Moodle Themes: Boost
+- Supported Moodle Version: 4.1 - 4.5
+- Supported PHP Version:    7.4 - 8.2
```

Comment:
Max PHP support would be 8.3 as we test against PHP 8.3 too

Reply:
[Open issue ](https://github.com/llttugraz/moodle-local_lytix/issues/1) is resolved in commits below.